### ISSUE TITLE
Interested in helping in documenting JuliaDSP

### ISSUE THREAD

**Issue by ashwani-rathee** [JuliaDSP/juliadsp.github.io#1]
@ViralBShah I am new to OSS contributions but I have contributed to Bokeh for documentation related issues ,jupyter notebooks improvements and theme inclusion..I am new to julia too(I did that coursera course on Julia Scientific Programming tho and it was pretty good)...I probably don't have right skillset yet but I want to contribute and learn..I made my first pull request to julia to JuliaMusic today only..

I want to help in maintaining and expanding the documentation and working on Julia's DSP related projects .

**Comment by ViralBShah** [comment]
Perhaps a juliadsp.org website?

**Comment by ashwani-rathee** [comment]
@ViralBShah That seems like a really good idea to me :)

**Comment by ViralBShah** [comment]
I think some folks in the JuliaDSP organization may have to chime in to help on that. I know @tlienart will be shortly announcing some tools to make it easy to put such websites together.

Many packages only have documentation, but the orgs do not yet have websites - since nobody has done it yet. Examples where there are websites are fluxml.ai and jump.dev. Those should serve as good starting points - gather the materials, packages, documentation, examples, videos and present them on the website.

**Comment by ashwani-rathee** [comment]
@ViralBShah I am thinking of using this Bulma Clean theme as a starting poiny for main organization website,both are using jekyll
https://github.com/chrisrhymes/bulma-clean-theme 
![ ](https://i.imgur.com/NRFFQ7H.png)
or
https://brave-submarine.cloudvent.net/
![ ](https://i.imgur.com/PU2FfnR.png)
or should it be kept more simple ??

**Comment by tlienart** [comment]
2 small notes: 

1. you could use both templates with Franklin (just a matter of putting  the right css in the  right  place)
2. PkgPage.jl which should be released this weekend, would help you build a nice looking landing page for a package or an org  in under 10 minutes.

I could help  you either way  depends how quickly you want this and whether you'd rather do that yourself with Jekyll or could be interested in using Franklin.

**Comment by ViralBShah** [comment]
I would prefer to use the PkgPage.jl template once it is released. That will make it easy to update and manage going forward.

**Comment by ashwani-rathee** [comment]
@tlienart @ViralBShah I would like to try Franklin and PkgPage.jl too
So till then the site can wait!! and I'll familiarize myself with franklin till then :)

**Comment by ashwani-rathee** [comment]
@ViralBShah @JuliaDSPteam I think we should start putting together the website for this place too

**Comment by ViralBShah** [comment]
Are there others here who can help brainstorm the content for a starter website for JuliaDSP?

**Comment by galenlynch** [comment]
Yeah I'd love to help brainstorm what would go on the website, I'm very excited that this is happening! @ssfrr and @martinholters were both maintainers of DSP.jl before I joined, and probably would have a lot of good ideas about what could go on the website.

I think it would be great to have some examples and cookbooks on the website, preferably some that tie together the different repos across the JuliaDSP organization. As it stands, the documentation for DSP.jl is anemic in terms of cookbooks or tutorials. I think just demonstrating how ones goes about doing basic tasks, such as designing and applying filters, doing spectral estimation, etc, would be amazing. I think Matlab actually does a really good job of this, and might be a useful point of comparison. Another good example is [LIGO's wonderful tutorial on detecting gravitational waves](https://www.gw-openscience.org/s/events/GW150914/GW150914_tutorial.html) ([or LIGO's other tutorials](https://www.gw-openscience.org/tutorials/)).

Maybe some benchmarks or something would be useful as well, as that seems like one of the possible "selling points" of Julia's DSP tooling over similar packages like SciPy.

The blog and links to a discourse channel, like fluxml.ai has, also seem good.

**Comment by galenlynch** [comment]
Also @rob-luke

**Comment by rob-luke** [comment]
This is a fantastic idea. I am glad someone has the skills and time to implement this. Thank you @ashwani-rathee 

> I think just demonstrating how ones goes about doing basic tasks, such as designing and applying filters, doing spectral estimation, etc, would be amazing.

Absolutely. I am sure we currently all have our own little snippets we use. We can all easily contribute these to a website once the skeleton structure is created.

>  Another good example is LIGO's wonderful tutorial on detecting gravitational waves

Yes these complete examples are always really handy. I have some similar full scale examples for other repos (but in python) that might be useful to see what features are handy to have in a website (auto links to docs, sidebars for text, dropdowns, etc) https://mne.tools/mne-nirs/auto_examples/plot_10_hrf.html

We had some discussion in the roadmap repo a while bag about merging a few of the DSP repos here. So I would just focus on DSP.jl

**Comment by ashwani-rathee** [comment]
Hey @rob-luke  @galenlynch  ,I was waiting for others in the team to respond to discuss on what the site should be like and It seems @ssfrr and @martinholters are currently unavailable ,I'll start making website and incorporating your ideas.. :)when the basic site is up,then we can start discussion on what needs to put up in the cookbook,examples section,blog.

**Comment by ashwani-rathee** [comment]
@galenlynch @rob-luke I will need around 20-25(for a start) examples with codes for visualizations(to be used in gallery,front page,tutorial section) showing the capabilites .

**Comment by ashwani-rathee** [comment]
And has there been discussion on the logo of the organization??It will be used everywhere so important to check

**Comment by galenlynch** [comment]
Sorry I've been such a slouch on this! I will provide some example this weekend. Thanks again for doing this! As for the logo, there organization has a badge that is three sinc waveforms (shown here: https://github.com/JuliaDSP), but maybe that could be improved upon? I'm not familiar with other discussion.
